nestkernel/node.h

```diff
@@ -477,6 +477,77 @@ class Node
   virtual void handle( DoubleDataEvent& e );
 
   /**
+   * Return the Ca_minus value at time t
+   * Return 0.0 if not overridden
+   */
+  virtual double_t
```

Comment:
I am confused about the `at time t` in the comment, given that the method does not take any time parameter. So I guess it means "at the present time". That makes me wonder who calls this method when. If nodes called it on each other during `update()`, this could lead to problems with causality given that the order of node updates is undefined, so that it is not clear what time it actually is. I also see potential problems in parallel simulations.

Could you comment on this and maybe clarify it in a comment? This also applies to other getter-methods below.